From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 27 Oct 2010 23:17:40 +0200
Subject: [PATCH] prepare for new autotools

we need AM_INIT_AUTOMAKE([foreign]) or automake fails with missing files

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 configure.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index e1941d008704..1f94ddeea03b 100755
--- a/configure.in
+++ b/configure.in
@@ -4,8 +4,8 @@ dnl     This file is an input file used by the GNU "autoconf" program to
 dnl     generate the file "configure", which is run during the build
 dnl     to configure the system for the local environment.
 
-AC_INIT
-AM_INIT_AUTOMAKE(fcgi, 2.4.0)
+AC_INIT([fcgi],[2.4.0],[])
+AM_INIT_AUTOMAKE([foreign])
 
 AM_CONFIG_HEADER(fcgi_config.h)
 
